test/routes.spec.js

```diff
@@ -8,6 +8,23 @@ const test_database = require('../index').test_database;
 chai.use(chaiHttp);
 
 describe('API Routes', () => {
+
+  before((done) => {
```

Comment:
Thank you for adding this into our tests.  This will help make them much easier to deal with later